src/audio/drc/drc_math_lib/asin/c_fixpt/main.c

```diff
@@ -9,6 +9,9 @@
 #include "main.h"
 #include "drc_asin_fixpt.h"
 
+/* Type Definitions */
+#include "stdint.h"
```

Comment:
This commit and the above one should be merged to the one above them?

Reply:
Yes, Correct, initially i missed adding in first time so made next commit.